### PR TITLE
Feature/tab routing api

### PR DESCRIPTION
**Pull request by tomek-ifm**

### CODE REVIEW

File: dev/index.tsx [hunk 1]
```diff
@@ -15,24 +15,25 @@
...
+  //   factory: () =>
+  //     new CatalogClient({
+  //       // apiOrigin:
+  //       discoveryApi: {
+  //         getBaseUrl: () => Promise.resolve('http://localhost:7000/catalog'),
+  //       },
+  //       // basePath: '',
+  //     }),
+  //   implements: catalogApiRef,
+  // })
```
Comment (dtuite) [inline dev/index.tsx]: If this doesn't work you can probably just remove the file?

**Review by iain-b — APPROVED** [review]

File: dev/index.tsx (hunk 1 above)
Comment (tomek-ifm) [reply dev/index.tsx]: I think they might update @backstage/dev-utils to v22 and standalone might start working again so maybe its work to keep it for a while

**Review by dtuite — APPROVED** [review]

File: dev/index.tsx (hunk 1 above)
Comment (dtuite) [reply dev/index.tsx]: Makes sense.

**Comment by timja** [comment]
It would be good to update the README too with new instructions for integrating

**Comment by tomek-ifm** [comment]
> It would be good to update the README too with new instructions for integrating

Just did it before merge :)

**Comment by timja** [comment]
> > It would be good to update the README too with new instructions for integrating
> 
> Just did it before merge :)

ah missed that, screenshots are a little outdated I assume? will look to pull this in soon 😄

**Comment by dtuite** [comment]
Hey @timja! We will work to update the screenshots and other marketing materials soon. You're dead right that they need a refresh.

We'd love to hear what you think if you try out the plugin. Please open issues for any improvements you would like to see.

**Pull request merged**